app/services/notify/notify-returns-email.presenter.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+/**
+ *
+ * @module NotifyEmailPresenter
+ */
+
+const { notifyTemplates } = require('../../lib/notify-templates.lib.js')
+
+/**
+ *
+ * @param recipient
+ * @param returnsPeriod
+ * @param referenceCode
+ */
+function go(recipient, returnsPeriod, referenceCode) {
```

Comment:
presenters make sense to format the data for the notify calls. 

The question would be should we do it before each request is made. Or format all the recipients into notifications. 

I am leaning to changing all in the recipients in the array into a something like notifications. As we have a clear delineation between a recipient and a notification. 

As we are using the existing DB table with a personalisation column this make sense.